tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/CountryDbConfiguration.cs

```diff
@@ -36,5 +36,7 @@ public void Configure(EntityTypeBuilder<Country> builder)
             .Ignore(p => p.Value)
             .Property(x => x.CountryId)
             .HasConversion<CountryCode2Converter>();
+        builder.OwnsOne(e => e.ArabicName).Ignore(p => p.Value)
```

Comment:
This is ok for now, we will need to understand how the DatabaseConfigurator and Generator will work for this case

Reply:
Ok